cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -154,7 +154,8 @@ func TestGetAPIServerCommand(t *testing.T) {
 			expected: []string{
 				"kube-apiserver",
 				"--insecure-port=0",
-				"--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota",
+				"--enable-admission-plugin=NodeRestriction",
+				"--disable-admission-plugin=PersistentVolumeLabel",
```

Comment:
If this is deprecated I don't know why we need to explicitly specify.

Reply:
@luxas can comment further i guess, i was just following the guide here:
https://github.com/kubernetes/kubeadm/issues/840

Reply:
It's still enabled by default which is a bug. Until that bug is fixed, let's leave it out here.

Reply:
I'm fine with defaulting off, but lets cross-link an issue.

Reply:
should i cross link to this?:
https://github.com/kubernetes/kubeadm/issues/840

Reply:
@neolit123 I'm thinking about the api-server deprecation default issue, should be in the main repo somewhere.

Reply:
i can't seem to find it. only found the PR that deprecated `admission-control`: https://github.com/kubernetes/kubernetes/pull/58123

Reply:
https://github.com/kubernetes/kubernetes/pull/64326 and https://github.com/kubernetes/kubernetes/issues/52617 and https://github.com/kubernetes/kubernetes/pull/52618